#goal definition

uint8 command_mask
uint8 TAKEOFF=0
uint8 LAND=1
uint8 FOLLOWPATH=2
uint8 QrCODE_LAND=3 
uint8 END_PATH = 4

int32 qr_code

float32 path_vel

nav_msgs/Path path
---
#result definition
int32 result
---
#feedback
int32 progress
